### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Leimeroth**

### CODE REVIEW

File: day_2/atomicrex/WorkshopPotentialEAM.ipynb [hunk 1]
```diff
@@ -144,16 +182,16 @@
     }
    ],
    "source": [
-    "data_pr = Project(\"../../datasets\")\n",
+    "data_pr = Project(\"../../datasets/imported_datasets/\")\n",
     "if len(data_pr.job_table()) == 0:\n",
-    "    data_pr.unpack(\"Cu_training_archive\")\n",
+    "    data_pr.unpack(\"../../datasets/Cu_training_archive\")\n",
     "data_pr.job_table()"
```
Comment (jan-janssen) [inline day_2/atomicrex/WorkshopPotentialEAM.ipynb]: I guess these changes are causing the crash

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/pyiron_contrib-feedstock/pull/11

File: day_2/01-atomicrex/WorkshopPotentialEAM.ipynb [hunk 2]
```diff
@@ -492,43 +595,218 @@
    "source": [
     "## Define the atom types of the potential\n",
     "job.input.atom_types.Cu = None\n",
-    "##\n",
-    "job.input.fit_algorithm = job.factories.algorithms.ar_lbfgs(max_iter=500)\n",
+    "## Limited number of steps for the workshop\n",
+    "job.input.fit_algorithm = job.factories.algorithms.ar_lbfgs(max_iter=2000)\n",
```
Comment (jan-janssen) [inline day_2/01-atomicrex/WorkshopPotentialEAM.ipynb]: @Leimeroth I saw you increased the number of steps from `500` to `2000` is this necessary? Already now the tests take more than twice as long and they are still running.

**Comment by Leimeroth** [comment]
Will change it back to 500. Also changed a few explanations.

**Comment by Leimeroth** [comment]
Seems to fail in another notebook should I merge it?

**Comment by jan-janssen** [comment]
> Seems to fail in another notebook should I merge it?

No, only merge pull requests which pass the tests.

**Pull request merged**